global/scripts/mppmodecheck.shlib

```diff
@@ -189,29 +28,19 @@ check_mpp_compliance()
       log_Msg "        the Extended MPP workflow is in active preparation and should be appropriately"
       log_Msg "        cited when published."
       log_Msg "        "
-      log_Msg "Checking available data and processing options"
-    else
-      log_Msg "HCPStyleData HCP Minimal Preprocessing Pipelines: ${pipeline} ${version}"
-      log_Msg "Checking data compliance with HCPStyleData HCP MPP mode"
     fi
 
-    check_mpp_${pipeline}
-
     # -- Final evaluation
 
     if [ "${MPPMode}" = "LegacyStyleData" ]; then
-      if [ "${Compliance}" = "LegacyStyleData" ]; then
-        log_Msg "Processing will continue using LegacyStyleData MPP mode."
+      if [ -z "${ComplianceMsg}" ]; then
```

Comment:
As I commented on a commit, do not use the existence of a message in the logic of whether to abort.  Message formatting is secondary, and should never replace the logic parameters.

Reply:
For reference: https://github.com/Washington-University/HCPpipelines/commit/2c4be159ccb44db54a0506b4d677b17b3ad87e1a#diff-85cc56a06f774b7c676d1160cade6f3bR402

Reply:
Changed.